projects/core/src/auth/user-auth/facade/auth.service.ts

```diff
@@ -87,10 +90,18 @@ export class AuthService {
    */
   async loginWithCredentials(userId: string, password: string): Promise<void> {
     try {
+      let uId: string = userId;
+
+      if (this.authMultisiteIsolationService?.getBaseSiteDecorator()) {
```

Comment:
Wouldn't it suffice to check just if the `this.authMultisiteIsolationService` is injected?
```ts
if(this.authMultisiteIsolationService){ ... }
```

Reply:
I think service can be injected, but if site is non isolated I would like to do nothing. So that's why I'm checking if the method returns any value.